app/model/commands/CreateWorkflowAtomCommand.scala

```diff
@@ -0,0 +1,89 @@
+package model.commands
+
+import java.util.Date
+import java.util.UUID._
+
+import com.gu.atom.data.IDConflictError
+import com.gu.contentatom.thrift.atom.media.{Category => ThriftCategory, MediaAtom => ThriftMediaAtom, Metadata => ThriftMetadata}
+import com.gu.contentatom.thrift.{Atom => ThriftAtom, _}
+import com.gu.media.logging.Logging
+import com.gu.pandomainauth.model.{User => PandaUser}
+import data.DataStores
+import model.commands.CommandExceptions._
+import model.{ChangeRecord, MediaAtom, WorkflowMediaAtom}
+
+import scala.util.{Failure, Success}
+
+case class CreateWorkflowAtomCommand(workflowMediaAtom: WorkflowMediaAtom, override val stores: DataStores, user: PandaUser)
+
+  extends Command with Logging {
+
+  type T = MediaAtom
+
+  def process(): MediaAtom = {
+    val atomId = randomUUID().toString
```

Comment:
> Nice urls: Some atoms could be rendered alone by frontend, and will use their id as the path (with a potential prefix such as `embed`). Having nice urls exactly like on article have several benefits.

Oooh! I like the sound of this. Currently, `media-atoms` can be associated with a video type Composer page to achieve this. For example here - https://www.theguardian.com/football/video/2017/may/19/new-wembley-arch-lit-up-team-colours-10th-anniversary-stadium-timelapse-video.

Also, does this slightly break the atom concept of being a part of content rather than content in their own right?